### DISCUSSION TITLE
服务器部署打不开

### DISCUSSION THREAD

**Discussion by Tyler913** [GaiZhenbiao/ChuanhuChatGPT#567] [General]
在家里的服务器上部署了，但是在自己电脑上用内网IP+端口号无法访问，这是为什么。在同一无线网局下

**Comment by oldfishoh** [comment]
> 在家里的服务器上部署了，但是在自己电脑上用内网IP+端口号无法访问，这是为什么。在同一无线网局下

配置文件里server_name那两行取消注释就好了

**Reply by Tyler913** [reply]
感谢，成功了
